source/wp-content/themes/wporg-themes-2024/src/theme-downloads/view.js

```diff
@@ -0,0 +1,82 @@
+/* global google, jQuery */
+
+async function renderChart( element ) {
+	const { theme = '', labelDate, labelValue } = element.dataset;
+	const url = 'https://api.wordpress.org/stats/themes/1.0/downloads.php?slug=' + theme + '&limit=260&callback=?';
+	const downloads = await jQuery.getJSON( url );
+
+	google.charts.setOnLoadCallback( function () {
+		const data = new google.visualization.DataTable();
+		let count = 0;
+
+		data.addColumn( 'string', labelDate );
+		data.addColumn( 'number', labelValue );
+
+		Object.entries( downloads ).forEach( ( [ key, value ] ) => {
+			data.addRow();
+			data.setValue( count, 0, new Date( key ).toLocaleDateString() );
+			data.setValue( count, 1, Number( value ) );
+			count++;
+		} );
+
+		new google.visualization.ColumnChart( element ).draw( data, {
+			title: 'Density of Precious Metals, in g/cm^3',
```

Comment:
Do we need this title? :)

Reply:
Nope! Whoops. I was trying out different settings and this comes from the chart docs 🤦🏻

Reply:
Removed in https://github.com/WordPress/wporg-theme-directory/pull/54/commits/8a107d47eaa8bbd697a47ddd13a1ab285dce43f9